notebooks/lesson06_model-interpretation.ipynb

```diff
@@ -0,0 +1,2088 @@
+{
```

Comment:
<p>I am not sure I fully follow the argument here.</p><p> </p><ol><li>we look how much predictions vary w.r.t. to a single feature.</li><li>if it predictions vary a lot this means the feature is not that predictive &lt;--&gt; low confidence?</li></ol><p>in a RF the decision could be chained like "if feature A AND feature B". example: feature "industrial area" and "close to ocean". the "close to ocean" feature might look like two distributions because of the "industrial area" feature and thus have a high variance while still being predictive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>the idea i was trying to convey is that if the confidence intervals are large then we have high variance =&gt; less predictive. i agree the wording may not be ideal, so i'll tweak it</p>